nltk/translate/bleu_score.py

```diff
@@ -58,12 +58,23 @@ def sentence_bleu(references, hypothesis, weights=(0.25, 0.25, 0.25, 0.25),
     >>> sentence_bleu([reference1, reference2, reference3], hypothesis1) # doctest: +ELLIPSIS
     0.5045...
 
+    If no ngram overlap exists for any of the orders that are used to evaluate BLEU
+    it will return the value zero. This since BLEU is a geometric mean of the precisions
+    for the different ngram orders (no overlap means a precision of 0).
+
```

Comment:
Maybe being a little verbose here would help ease the users' pain when this is merged:

> If there are no ngrams overlaps for any order of n-grams, BLEU would return the value 0. 
> This is because the precision for the order of n-grams without overlap returns a 0 and
> the geometric mean in the final BLEU score computation multiplies the 0 with the 
> precision of of other n-grams, which results in 0.

Reply:
I changed to what you proposed now.

I also expanded the description of the test below. And updated it (origianlly it said 6-grams but had 5 weights). Now it says 5grams and have 5 weights.